spec/features/asylum_and_immigration_spec.rb

```diff
@@ -10,7 +9,7 @@
       visit_flow_page(target: :level_of_help, passporting: true)
       select_radio(page:, form: "level-of-help-form", field: "level-of-help", value: "controlled")
       click_on "Save and continue"
-      expect(page).not_to have_content tribunal_header
+      expect(page).not_to have_content matter_type_header
```

Comment:
this test would still have passed w/o this change. Should we make it a positive test instead?